.github/actions/sync-nightlies/action.yml

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Since this is not source files isn't the SPDX header format enough here (i.e. `SPDX-License-Identifier: Apache-2.0`)?

Reply:
this is a copy of a file checked into other pekko repos - it just seems easier to keep the same contents